Razor/Scripts/Outlands.cs

```diff
@@ -342,5 +343,32 @@ private static int Hue(string expression, Variable[] args, bool quiet)
 
             return item.Hue;
         }
+
+        private static bool Paralyzed(string expression, Variable[] args, bool quiet)
+        {
+            // Simple expresion for Paralyzed
+            if (World.Player == null)
+                return false;
+
```

Comment:
Is there a Paralyzed flag on the player? Paralyze isn't necessarily sent as a buff in all cases. It's a special flag in a packet.

Reply:
removed sinct paralyze go to buffexist

Reply:
Ok, i found them now - introduce based on Flag 0x1